exporter/collector.go

```diff
@@ -63,6 +64,42 @@ func (mc JSONMetricCollector) Collect(ch chan<- prometheus.Metric) {
 				level.Error(mc.Logger).Log("msg", "Failed to convert extracted value to float64", "path", m.KeyJSONPath, "value", value, "err", err, "metric", m.Desc)
 				continue
 			}
+		} else if m.Mincount > 0 { // ScrapeType is 'countbylabel'
```

Comment:
Hmm, this side-effect pattern is bothering me. We should really have `JSONMetric{}` have an explicit `Type` passed to it.